sdk/keyvault/azure-keyvault-keys/pyproject.toml

```diff
@@ -20,26 +25,35 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
-    "Programming Language :: Python :: 3.13",
     "License :: OSI Approved :: MIT License",
 ]
+requires-python = ">=3.9"
+keywords = ["azure", "azure sdk"]
+
 dependencies = [
-    "azure-core>=1.31.0",
-    "cryptography>=2.1.4",
     "isodate>=0.6.1",
+    "azure-core>=1.31.0",
     "typing-extensions>=4.6.0",
+    "cryptography>=2.1.4",
+]
+dynamic = [
+"version", "readme"
 ]
-dynamic = ["version", "readme"]
 
 [project.urls]
 repository = "https://github.com/Azure/azure-sdk-for-python/tree/main/sdk"
 
 [tool.setuptools.dynamic]
-version = {attr = "azure.keyvault.keys._version.VERSION"}
+version = {attr = "azure.keyvault.keys._generated._version.VERSION"}
```

Comment:
Manually updated this path from: `azure.keyvault.keys._generated._version.VERSION` to account for convenience layer manual changes: https://gist.github.com/mccoyp/1df65df4102d158c7b75aee4b4b173cc#update-the-convenience-layer